actors/builtin/market/policy.go

```diff
@@ -7,7 +7,7 @@ const DealUpdatesInterval = 100
 
 // Bounds (inclusive) on deal duration
 func dealDurationBounds(size abi.PaddedPieceSize) (min abi.ChainEpoch, max abi.ChainEpoch) {
-	return abi.ChainEpoch(0), abi.ChainEpoch(10000) // PARAM_FINISH
+	return abi.ChainEpoch(0), abi.ChainEpoch(abi.SecondsInYear / abi.EpochDurationSeconds) // min 0, max 1y, PARAM_FINISH
```

Comment:
Why 1 year? Why is there a max? Why not a millenium? If there's some concrete justification please write it here in a comment, otherwise use a "never going to happen" large number.

Reply:
+1

Reply:
we are expecting to upgrade PoRep fairly often anyways, a year seems fine? We dont have the feature to extend deals yet

Reply:
I think it's more a question of what the number "means" -- I feel like the only good numbers for placeholders are 0 and infinity (regardless of whether our schedule means something like 1 year is "practically infinity").

Reply:
It’s not just a placeholder, it’s an assumption we have been making for a while in cryptoecon that we only consider sector lifetimes up to 1 year. After 1 year it may become rational to terminate (probably not, but I‘d want to think about it more if we want to increase this bound).

Reply:
Ok, I will document that assumption in the code.

Note that this deal bound does not bound sector lifetimes, though.